source/CILobbyMenu.cpp

```diff
@@ -171,25 +171,16 @@ void LobbyMenu::update(MenuState& state) {
     {
         case MenuState::MainToLobby:
         {
-            // handle displaying for Host
+            // handle game id display 
             _gameSettings->setGameId(_networkMessageManager->getRoomId());
-            setPlayerLabels({ "N/A", "N/A", "N/A", "N/A", "N/A" });
-
-            _lobbyRoomLabel->setText(_gameSettings->getGameId());
-
-            setDisplay(true);
-
-            state = _nextState = MenuState::GameLobby;
-            break;
+            [[fallthrough]];
         }
         case MenuState::JoinToLobby:
         {
             // handle setting up the lobby
-            setPlayerLabels({ "N/A", "N/A", "N/A", "N/A", "N/A" });
+            setPlayerLabels({ "Waiting...", "Waiting...", "Waiting...", "Waiting...", "Waiting..." });
```

Comment:
I actually think having N/A is better because waiting seems like you are waiting for 4 other players when there could be less.